### PR TITLE
Add alerts for dabealu/zookeeper-exporter

### PR DESCRIPTION
**Pull request by strangeman**

### CODE REVIEW

File: _data/rules.yml [hunk 1]
```diff
@@ -1090,6 +1090,29 @@ groups:
...
+                for: 3m
+              - name: Zookeeper Too Many Leaders
+                description: "Zookeeper cluster has too many nodes marked as leader"
+                query: 'sum(zk_server_leader) > 1'
+                severity: critical
+                for: 3m
+              - name: Zookeeper Not Ok
+                description: "Zookeeper instance is not ok"
+                query: 'zk_ruok == 0'
+                severity: high
```
Comment (samber) [inline _data/rules.yml:1114]: warning

File: _data/rules.yml [hunk 2]
```diff
@@ -1090,6 +1090,29 @@ groups:
...
             doc_url: https://github.com/cloudflare/kafka_zookeeper_exporter
             rules:
+          - name:  dabealu/zookeeper-exporter
+            doc_url: https://github.com/dabealu/zookeeper-exporter
+            rules:
+              - name: Zookeeper Down
+                description: "Zookeeper down on instance {{ $labels.instance }}"
+                query: 'zk_up == 0'
+                severity: critical
+                for: 3m
```
Comment (samber) [inline _data/rules.yml:1100]: You can remove the `for` clause. It will fire immediately.

File: _data/rules.yml [hunk 3]
```diff
@@ -1090,6 +1090,29 @@ groups:
...
+              - name: Zookeeper missing leader
+                description:  "Zookeeper cluster has no node marked as leader"
+                query: 'sum(zk_server_leader) == 0'
+                severity: critical
+                for: 3m
```
Comment (samber) [inline _data/rules.yml:1105]: Same

File: _data/rules.yml [hunk 4]
```diff
@@ -1090,6 +1090,29 @@ groups:
...
+              - name: Zookeeper Too Many Leaders
+                description: "Zookeeper cluster has too many nodes marked as leader"
+                query: 'sum(zk_server_leader) > 1'
+                severity: critical
```
Comment (samber) [inline _data/rules.yml:1109]: Same

**Review by samber — APPROVED** [review]
Hi @strangeman and thanks for this contribution :tada:

I made some comments ;)

File: _data/rules.yml (hunk 2 above)
Comment (strangeman) [reply _data/rules.yml:1100]: Hmm, in my opinion, immediate alert firing can lead to some false alarms and alert fatigue. Maybe just decrease it to `1m` to handle planned Zk restarts, for example?

Comment (samber) [reply _data/rules.yml:1100]: In that case, we are not alerting for a container crash, but for a critical infrastructure component.

If ZK is not able to restart by itself ASAP, I think you can ping Ops.

WDYT ?

Comment (strangeman) [reply _data/rules.yml:1100]: It depends on real scenarios (e.g., in our installation, losing one Zk node is not crucial), but your argument has good reasons. Anyway, it is just examples, and they should be adapted for real cases. :)

Comment (strangeman) [reply _data/rules.yml]: fixed

File: _data/rules.yml (hunk 3 above)
Comment (strangeman) [reply _data/rules.yml]: fixed

File: _data/rules.yml (hunk 4 above)
Comment (strangeman) [reply _data/rules.yml:1107]: fixed

File: _data/rules.yml (hunk 1 above)
Comment (strangeman) [reply _data/rules.yml]: fixed

**Review by samber — APPROVED** [review]

**Comment by samber** [comment]
Look's good ;)

Thanks for your help!

**Pull request merged**